lms/templates/vert_module.html

```diff
@@ -1,4 +1,8 @@
 <%page expression_filter="h"/>
+<%! from openedx.core.djangolib.markup import HTML %>
+%if unit_title:
+  <h3 class="unit-title">${HTML(unit_title)}</h3>
```

Comment:
@alisan617 with the expression_filter in place, all variables are escaped automatically. So this can just be:

    <h3 class="unit-title">${unit_title}</h3>